M/MKL_Headers/build_tarballs.jl

```diff
@@ -3,81 +3,48 @@
-    ),
-    ArchiveSource(
-        "https://conda.anaconda.org/intel/linux-64/mkl-devel-2024.2.0-intel_663.tar.bz2",
-        "e3c37c75aa870aa8daa32e6cbfa6e34639f7e6fe6a67fc4b34fa2a94a497df15";
-        unpack_target = "mkl-devel-x86_64-linux-gnu"
-    ),
+    FileSource("https://files.pythonhosted.org/packages/cd/b7/092df7c41f57b3a41bb60392081564da0ea64ae300128fa9d44dd7acd84b/mkl_include-2025.0.1-py2.py3-none-win_amd64.whl",
+               "1149c1b34bc41166c0bb09c103e42ae251f0a9d190b886a99124b70632b3bc8d"; filename="mkl_include-x86_64-w64-mingw32.whl"),
+    FileSource("https://files.pythonhosted.org/packages/7b/42/74f3ed839c59e7bf3992804ce172590b4acac01dac4157f81daeb7774202/mkl_include-2025.0.1-py2.py3-none-manylinux_2_28_x86_64.whl",
+               "a70b90ce07f7a970a6fc8b324e416e0e978edec751b2d2e1b52d511f886fe506"; filename="mkl_include-x86_64-linux-gnu.whl"),
+    FileSource("https://files.pythonhosted.org/packages/1f/b1/03a2388889052dd018cebf8b915599015161519c6af89e42e4a31a62e2f0/mkl_devel-2025.0.1-py2.py3-none-win_amd64.whl",
+               "fc62df9c689722dadf282af6380538063dbd4fd3d328aa3c60eb72a521d06789"; filename="mkl_devel-x86_64-w64-mingw32.whl"),
+    FileSource("https://files.pythonhosted.org/packages/fd/e1/b13109f2d31fc6b1e753d0746317a62a119292ec127e196b409ad420587b/mkl_devel-2025.0.1-py2.py3-none-manylinux_2_28_x86_64.whl",
+               "18662a91ce12613622b7e002b7f5dcb790bf22bba83cef80179b9d2d3458ed2d"; filename="mkl_devel-x86_64-linux-gnu.whl"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/mkl-include-$target
+cd $WORKSPACE/srcdir
+unzip -d mkl_include-$target mkl_include-$target.whl
+unzip -d mkl_devel-$target mkl_devel-$target.whl
+
+cd $WORKSPACE/srcdir/mkl_include-$target
 if [[ $target == *-mingw* ]]; then
-    rsync -av Library/include/ ${includedir}
+    rsync -av mkl_include-${target}/mkl_include-2025.0.1.data/data/Library/include/ ${includedir}
 else
-    rsync -av include/ ${includedir}
+    rsync -av mkl_include-${target}/mkl_include-2025.0.1.data/data/include/ ${includedir}
 fi
-install_license info/licenses/*.txt
+install_license mkl_include-${target}/mkl_include-2025.0.1.dist-info/LICENSE.txt
 
-cd $WORKSPACE/srcdir/mkl-devel-$target
+cd $WORKSPACE/srcdir/mkl_devel-$target
 mkdir -p ${libdir}
 if [[ $target == *-mingw* ]]; then
     # These toolchain files must still go inside the lib folder, not the ${libdir} folder
-    rsync -av Library/lib/ $WORKSPACE/destdir/lib
+    rsync -av mkl_devel-${target}/mkl_devel-2025.0.1.data/data/Library/lib/ $WORKSPACE/destdir/lib
 else
-    rsync -av lib/ ${libdir}
+    rsync -av mkl_devel-${target}/mkl_devel-2025.0.1.data/data/lib/ ${libdir}
 fi
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
     Platform("x86_64", "linux"; libc="glibc"),
-    Platform("i686", "linux"; libc="glibc"),
-    Platform("i686", "windows"),
```

Comment:
For posterity, the removal of this was hidden in the release notes for the full toolkit here https://www.intel.com/content/www/us/en/developer/articles/release-notes/intel-oneapi-toolkit-release-notes.html#inpage-nav-6, and is not actually mentioned on any of the individual tool pages.